articles/hdinsight/hbase/apache-hbase-using-phoenix-query-server-rest-sdk.md

```diff
@@ -1,538 +1,538 @@
-    {
-        await client.CloseStatementRequestAsync(connId, statementHandle.Id, options);
-        statementHandle = null;
-    }
-    if (openConnResponse != null)
-    {
-        await client.CloseConnectionRequestAsync(connId, options);
-        openConnResponse = null;
-    }
-}
-```
-
-The output of the `select` statements should be the following result:
-
-```
-id0 first0
-id1 first1
-id10 first10
-id100 first100
-id101 first101
-id102 first102
-. . .
-id185 first185
-id186 first186
-id187 first187
-id188 first188
-
-Number of rows: 100
-Frame row count: 500
-Fetch response is done: True
-
-Total customer records: 600
-
-NJ: 21
-CA: 19
-GU: 17
-NC: 16
-IN: 16
-MA: 16
-AZ: 16
-ME: 16
-IL: 15
-OR: 15
-. . . 
-MO: 10
-HI: 10
-GA: 10
-DC: 9
-NM: 9
-MD: 9
-MP: 9
-SC: 7
-AR: 7
-MH: 6
-FM: 5
-```
-
-## Next steps 
-
-* [Phoenix in HDInsight](../hdinsight-phoenix-in-hdinsight.md)
-* [Using the HBase REST SDK](apache-hbase-rest-sdk.md)
+---
+title: Phoenix Query Server REST SDK - Azure HDInsight 
+description: Install and use the REST SDK for the Phoenix Query Server in Azure HDInsight.
+services: hdinsight
+ms.service: hdinsight
+author: ashishthaps
+ms.author: ashishth
+ms.reviewer: jasonh
+ms.custom: hdinsightactive
+ms.topic: conceptual
+ms.date: 12/04/2017
+---
+# Apache Phoenix Query Server REST SDK
+
+[Apache Phoenix](http://phoenix.apache.org/) is an open source, massively parallel relational database layer on top of [Apache HBase](apache-hbase-overview.md). Phoenix allows you to use SQL-like queries with HBase through SSH tools such as [SQLLine](apache-hbase-phoenix-squirrel-linux.md). Phoenix also provides an HTTP server called Phoenix Query Server (PQS), a thin client that supports two transport mechanisms for client communication: JSON and Protocol Buffers. Protocol Buffers is the default mechanism, and offers more efficient communication than JSON.
+
+This article describes how to use the PQS REST SDK to create tables, upsert rows individually and in bulk, and select data using SQL statements. The examples use the [Microsoft .NET driver for Apache Phoenix Query Server](https://www.nuget.org/packages/Microsoft.Phoenix.Client). This SDK is built on [Apache Calcite's Avatica](https://calcite.apache.org/avatica/) APIs, which exclusively use Protocol Buffers for the serialization format.
+
+For more information, see [Apache Calcite Avatica Protocol Buffers Reference](https://calcite.apache.org/avatica/docs/protobuf_reference.html).
+
+## Install the SDK
+
+Microsoft .NET driver for Apache Phoenix Query Server is provided as a NuGet package, which can be installed from the Visual Studio **NuGet Package Manager Console** with the following command:
+
+    Install-Package Microsoft.Phoenix.Client
+
+## Instantiate new PhoenixClient object
+
+To begin using the library, instantiate a new `PhoenixClient` object, passing in `ClusterCredentials` containing the `Uri` to your cluster and the cluster's Apache Hadoop user name and password.

```

Comment:
The portal now calls this user name the Cluster login username and Cluster login password. 
![image](https://user-images.githubusercontent.com/5067358/49033742-d35e5b00-f164-11e8-9093-dd2bffa0075d.png)